jkube-kit/doc/src/main/asciidoc/inc/_generator.adoc

```diff
@@ -71,11 +75,12 @@ include::generator/_webapp.adoc[]
 include::generator/_quarkus.adoc[]
 include::generator/_openliberty.adoc[]
 include::generator/_micronaut.adoc[]
-
-include::generator/_api.adoc[]
 ifeval::["{plugin-type}" == "maven"]
 include::generator/_karaf.adoc[]
 include::generator/_wildfly_jar.adoc[]
+include::generator/_helidon.adoc[]
```

Comment:
Doesn't the generator work on Gradle?

Reply:
Helidon doesn't support Native Image mode in gradle.

Reply:
When I reviewed your PRs from yesterday, I saw that there were tests that contained a similar project to: https://github.com/helidon-io/helidon/blob/2.6.0/examples/quickstarts/helidon-quickstart-mp/build.gradle

The generator doc contains entries for gradle (within this same PR). So again, I'm not sure if we are supporting or not Gradle. If not, then everything is pretty inconsistent.

> Even [Helidon Starter](https://helidon.io/starter/3.2.1?step=1) support maven only at the moment.

This shouldn't be relevant. It should be relevant if Helidon works on Gradle or not.
- https://github.com/helidon-io/helidon/blob/a20fe937749cf1b42701a24b7de4ad941b738cba/examples/quickstarts/helidon-quickstart-se/build.gradle
- https://github.com/helidon-io/helidon/blob/a20fe937749cf1b42701a24b7de4ad941b738cba/examples/quickstarts/helidon-quickstart-mp/build.gradle

Have you tried these? could you clarify if they work or not.

Reply:
Ok, but it does work in standard JVM mode, so we should add it nonetheless
Also we need to keep this in mind. We can have a better UX by warning users that Native mode is not applicable in gradle. so probably an improvement for the generator

Reply:
I've moved generator and healthcheck enricher to gradle